test/unit/math/prim/prob/ordered_probit_test.cpp

```diff
@@ -17,6 +17,46 @@ stan::math::vector_d get_simplex_Phi(double lambda,
   return theta;
 }
 
+double old_ordered_probit(int y, double lambda, stan::math::vector_d c) {
+  using stan::math::log;
+  using stan::math::log1m;
+  using stan::math::Phi;
+
+  int K = c.size() + 1;
+
+  if (y == 1) {
+    return log1m(Phi(lambda - c(0)));
+  } else if (y == K) {
+    return log(Phi(lambda - c(K - 2)));
+  } else {
+    return log(Phi(lambda - c(y - 2)) - Phi(lambda - c(y - 1)));
+  }
+}
```

Comment:
The problem with testing old code doesn't work is that isn't something we want to guarantee. If we replace `Phi` with `normal_lcdf` (and we probably should if the `normal_lcdf` numerics are better) then this code would probably start working and the test breaks.

Just as long as the tests prevent us from reintroducing the bug is good enough. It's kinda on the reviewer to do a little manual checking to make sure the old code was actually failing the tests (not something that needs included).

So delete the `old_ordered_probit` stuff. Apologies for not being clear originally on what more testing meant.

Reply:
@andrjohns actually I just went ahead and pushed to your branch. So all you gotta tell me is thumbs up on my changes :D. I approve of this as is.

Reply:
Huh that does save time, all good with me!

> If we replace Phi with normal_lcdf (and we probably should if the normal_lcdf numerics are better) then this code would probably start working and the test breaks.

In that case, would it be simpler if I included in this pull:

 - ```Phi``` redefined as ```exp(std_normal_lcdf)```
 - ```log_Phi``` introduced as an alias for ```std_normal_lcdf```
?

Or leave as-is and do in a separate pull?

Reply:
Separate pull, and only if you want to do it.

A more stable `Phi` and making `normal_lccdf` use the same numerics as lcdf like @nhuurre pointed out would be good. It's probably worth checking `std_normal_lcdf` and `std_normal_lccdf` as well.

No need to add a `log_Phi`. Got enough functions as is :D.